app/views/providers/means_reports/show.html.erb

```diff
@@ -9,9 +9,11 @@
 
   <%= render(
         'shared/check_answers/client_details',
-        attributes: %i[first_name last_name date_of_birth age national_insurance_number],
+        attributes: %i[first_name last_name date_of_birth age means_test national_insurance_number],
         applicant: @legal_aid_application.applicant,
         read_only: true) %>
 
-  <%= render "with_cfe_result_details" %>
+  <% if !@legal_aid_application.non_means_tested? %>
```

Comment:
The double negative here is a little confusing to me.

I wonder whether it would be more readable to say:

```erb
<% unless @legal_aid_application.non_means_tested? %>
```

Or perhaps if we defined another predicate method, something like this might work:

```erb
<% if @legal_aid_application.means_tested? %>
```

What do you think?